src/Button/Button.tsx

```diff
@@ -147,6 +151,11 @@ function Button<P extends React.ElementType = "button">(
       {icon && (
         <>
           {icon}
+          {iconText && (
```

Comment:
I'm wondering if we can do this in Icon instead... thinking. We need to refactor the buttons in general and have a look at what we have, how we use them and decide a more final API for them